### DISCUSSION TITLE
How to enable clustering feature only for one specific job?

### DISCUSSION THREAD

**Discussion by haskic** [quartznet/quartznet#2015] [Q&A] [answered]
Does the clustering feature affect all jobs?
I have 2 jobs: 1st job should be executed only by the one instance (I'm using the clustering feature for this) and 2nd job should be executed by all instances. How to implement such behavior? Thank you!

**Comment by lahma** [comment, marked as answer]
You will need to schedulers then. Clustering is a per-scheduler feature, but you can run multiple schedulers.
